external/fv3fit/tests/test_squashed_output_model.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import xarray as xr
+import pytest
+from fv3fit._shared.models import SquashedOutputModel
+from fv3fit.testing import ConstantOutputPredictor
+
+
+ARRAY = np.array([[-2.0, -1.0, 0.0, 1.0, 2.0]])
+OUTPUT_DICT = {"a": ARRAY, "b": 0.1 * ARRAY}
+
+
+@pytest.mark.parametrize(
+    ["squash_to", "squash_threshold", "expected"],
+    [
+        pytest.param(None, None, OUTPUT_DICT, id="no squash"),
+        pytest.param(
+            0.0,
+            1.5,
+            {"a": [[0.0, 0.0, 0.0, 0.0, 2.0]], "b": [[0.0, 0.0, 0.0, 0.0, 0.2]]},
+            id="1.5_to_zero",
+        ),
+        pytest.param(
+            0.0,
+            -1.5,
+            {"a": [[0.0, -1.0, 0.0, 1.0, 2.0]], "b": [[0.0, -0.1, 0.0, 0.1, 0.2]]},
```

Comment:
The original `b` array has -0.2 as its first element, shouldn't that remain as is and not get squashed to 0.0? I could also be just missing something.

Reply:
The idea is that the squashing for all variables is determined by the values in the `squash_by_name` field (here "a", which is less than the threshold, so "b" gets squashed). Since this seems not very clear, I think it's a good idea to move to configuring both the `squash_by_name` and `squash_target_names` explicitly.